dockerfiles/Dockerfile.apisix-base.apk

```diff
@@ -54,23 +54,6 @@ RUN apk add --no-cache \
 # install latest Rust to build wasmtime
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 
-RUN cd /tmp \
```

Comment:
We no longer maintain the alpine image, you don't need to modify this.
Of course, you can remove it directly in another PR

Reply:
The test that was failing uses this image. I have modified it to install openssl3